### ISSUE TITLE
Support for creating posts? 

### ISSUE THREAD

**Issue by JVT038** [burhan-syed/troddit#13]
Hi,
I'm currently testing Troddit, and I like it so far.
Only thing that misses, is the ability to create text, video, photo, and just normal posts.
Or perhaps I've missed it somewhere in the UI?

**Comment by burhan-syed** [comment]
Currently it's read only with some interaction with votes/replies. Post creation is a project yet to be started.

**Comment by uAliFurkanY** [comment]
I'm not making promises here but if I coded it would you accept a PR?

**Comment by burhan-syed** [comment]
@uAliFurkanY yes of course. It should open a submission page on /submit or r/[subreddit]/submit. Subreddit submission requirements should be checked prior to submitting. 
Of course link and text posts should be supported but preferably you'd also be able to submit video and image posts directly as on Reddit. From what I've seen the latter two are not very well documented but should be possible.

**Comment by Prayag2** [comment]
Any updates on this?

**Comment by uAliFurkanY** [comment]
i've given up, i hate react
